notebooks/confluent-cloud-integration/notebook.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "f08deb45-4350-49f4-8fc7-643f73e21413",
+      "metadata": {},
+      "source": [
+        "<div id=\"singlestore-header\" style=\"display: flex; background-color: rgba(235, 249, 245, 0.25); padding: 5px;\">\n",
+        "    <div id=\"icon-image\" style=\"width: 90px; height: 90px;\">\n",
+        "        <img width=\"100%\" height=\"100%\" src=\"https://raw.githubusercontent.com/singlestore-labs/spaces-notebooks/master/common/images/header-icons/database.png\" />\n",
+        "    </div>\n",
+        "    <div id=\"text\" style=\"padding: 5px; margin-left: 10px;\">\n",
+        "        <div id=\"badge\" style=\"display: inline-block; background-color: rgba(0, 0, 0, 0.15); border-radius: 4px; padding: 4px 8px; align-items: center; margin-top: 6px; margin-bottom: -2px; font-size: 80%\">SingleStore Notebooks</div>\n",
+        "        <h1 style=\"font-weight: 500; margin: 8px 0 0 4px;\">Ingest data from Confluent Cloud (Kafka)</h1>\n",
+        "    </div>\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Ingest data from Confluent Cloud (Kafka)"
+      ],
+      "metadata": {},
+      "id": "bfd1a1969826a2e7"
+    },
```

Comment:
Can you remove this title block? The title gets put into the header above automatically.

Reply:
done

Reply:
I think you may have removed the wrong one. It looks like the generated header and footer are gone now.

Reply:
If you run the pre-commit checks again I think that should fix it.

Reply:
Unable to run, I always got error: 

Traceback (most recent call last):
  File "/home/oyeliseiev-ua/Singlestore/Repositories/spaces-notebooks/resources/toml-check.py", line 5, in <module>
    import tomllib

I'm not experienced with python to fix it quickly

Reply:
You must have been able to do it once. The first version of your ipynb file was correct except for the addition of an extra header. If you go back to that version, I'll approve it and just fix that one issue after it's merged.

Reply:
this should be resolved, @kesmit13 could you please check?